### PR TITLE
Fix coding style via StyleCI

### PR DESCRIPTION
**Pull request by peter279k**
As title. Fix coding style via StyleCI service.

### CODE REVIEW

File: bin/root-version.php [hunk 1]
```diff
@@ -34,23 +35,11 @@ function get_last_tag_name(): string
 
     if (JSON_ERROR_NONE !== json_last_error()) {
         // TODO: switch to safe json parsing in the future
-        throw new RuntimeException(
```
Comment (theofidry) [inline bin/root-version.php:37]: This is one is not desired IMO. I think it's inherited from the Symfony style, but IMO it only hurts readability here

File: src/PhpParser/NodeVisitor/StringScalarPrefixer.php [hunk 2]
```diff
@@ -131,12 +131,12 @@ private function prefixStringScalar(String_ $string): String_
...
         if (false === (
-                $parentNode instanceof Assign
+            $parentNode instanceof Assign
                 || $parentNode instanceof Param
                 || $parentNode instanceof Const_
                 || $parentNode instanceof PropertyProperty
                 || $parentNode instanceof Return_
-            )
+        )
         ) {
```
Comment (theofidry) [inline src/PhpParser/NodeVisitor/StringScalarPrefixer.php:140]:
```suggestion
        )) {
```

Comment (peter279k) [reply src/PhpParser/NodeVisitor/StringScalarPrefixer.php]: Fixed.

File: bin/root-version.php (hunk 1 above)
Comment (peter279k) [reply bin/root-version.php:37]: @theofidry, thanks for your reply.

Actually, I also agree with you, do you have any suggestion to let Style CI analyze the coding style successfully if revering these style changes?

Comment (theofidry) [reply bin/root-version.php:37]: Should be configurable with `.styleci.yml`, as to what parameter exactly I do not know

Comment (peter279k) [reply bin/root-version.php]: @theofidry, thanks for your reply.

I've disabled the `single_line_throw` setting on `.styleci.yml` setting to ignore this.

**Comment by theofidry** [comment]
Thanks!

**Pull request merged**